### PR TITLE
Switch import formatting to use Ruff

### PR DESCRIPTION
**Pull request by JacobCallahan**
This now drops the old standard of one import per line in favor of a less noisy standard (isort).

### CODE REVIEW

File: pyproject.toml
Comment (ogajduse) [inline pyproject.toml:1]: What do you think about keeping the isort config the same as in broker?
https://github.com/SatelliteQE/broker/blob/dd2471729379f06cb7c809dfbbc116ed5e5a91e7/pyproject.toml#L174-L179

Comment (ogajduse) [reply pyproject.toml:1]: I also see that I duplicated the `tool.black` section. Can you please remove it?

Comment (JacobCallahan) [reply pyproject.toml:1]: good point. will update and see what happens

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo

**Review by ogajduse — APPROVED** [review]
PRT passed :heavy_check_mark:

**Review by omkarkhatavkar — APPROVED** [review]

**Pull request merged**